infra/build/functions/target_experiment.py

```diff
@@ -74,8 +75,11 @@ def run_experiment(project_name, target_name, args, output_path,
   local_output_path = '/workspace/output.log'
   local_corpus_path_base = '/workspace/corpus'
   local_corpus_path = os.path.join(local_corpus_path_base, target_name)
+  local_target_path = os.path.join(build.out, target_name)
```

Comment:
This might not be the case for all builds, since some targets may exist in subdirs. Let's skip uploading the binary for now

Reply:
Given we know the binary name, can I use `find` as a heuristic to find this path?

Reply:
Yes, that might work, if it's not too difficult to do this within a Cloud Build step.

Reply:
Added the script for this.
The script uses the this path as the default, and use `find` to locate target binary if it is not here.